elliottlib/bzutil.py

```diff
@@ -38,17 +68,15 @@ def search_for_bugs(bz_data, status, verbose=False):
     for r in bz_data.get('target_release'):
         query_url.addTargetRelease(r)
 
-    # TODO: Expose this for debugging
-    if verbose:
-        click.echo(query_url)
+    return query_url
+
+def _perform_query(bzapi, query_url, include_fields=['id']):
```

Comment:
not supposed to put collections in default params... default to `None` or `Missing` and set the default accordingly

`query["include_fields"] = include_fields if include_fields is not None else ['id']`

Reply:
Fixed.